langkit/textstat.py

```diff
@@ -62,31 +65,43 @@ def wrappee(text: Union[pd.DataFrame, Dict[str, List]]) -> Union[pd.Series, List
     return wrappee
 
 
-def init(config: Optional[LangKitConfig] = None):
-    pass
-
-
-init()
```

Comment:
The removal of the init() call in the import of these modules: (textstat, sentiment, regexes, etc) is a significant breaking change in behavior and will break existing LangKit integrations that imported these modules directly.

Can we provide a back-compat friendly version of the language support changes?